src/dom_cfg/rpi_5_domd.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2024 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <domain.h>
+#include <zephyr/xen/public/domctl.h>
+
+#include <string.h>
+
+static char *domd_dtdevs[] = {
+};
+
+static struct xen_domain_iomem domd_iomems[] = {
+	{ .first_mfn = 0x107D517, .nr_mfns = 0x1, .first_gfn = 0},
```

Comment:
Hm, this bothers me. Are you mapping MFN  `0x107D517` to GFN `0x0`?

Reply:
gfn=0 - means 1:1 mapping